### PR TITLE
Feature product flavour latest

### PR DESCRIPTION
**Pull request by sw-tt-tusharojha**

### CODE REVIEW

File: IOTConnectSDK/iotconnect-sdk/build.gradle [hunk 1]
```diff
@@ -62,8 +62,8 @@ afterEvaluate {
                 // You can then customize attributes of the publication as shown below.
                 groupId = 'com.github.sw-ms-imranbhadelia'
                 artifactId = 'iotconnect-sdk'
-                version = '3.1.3'
+                version = '30.0.0'
```
Comment (sw-ot-deveshmevada) [inline IOTConnectSDK/iotconnect-sdk/build.gradle]: Why there is a big difference between 3.1.3 and 30.0.0?

File: IOTConnectSDK/iotconnect-sdk_2_1/build.gradle [hunk 2]
```diff
@@ -71,4 +95,33 @@ dependencies {
...
+            brokerazRelease(MavenPublication) {
+                from components.brokerazRelease
+
+                groupId = 'com.github.sw-ms-imranbhadelia'
+                artifactId = 'iotconnect-sdk'
+                version = '12.2.1-azTest'
+
+            }
+
+            /*brokerawsRelease(MavenPublication) {
```
Comment (sw-ot-deveshmevada) [inline IOTConnectSDK/iotconnect-sdk_2_1/build.gradle:110]: remove unwanted code

File: IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/SDKClientManager.kt [hunk 3]
```diff
@@ -229,14 +233,15 @@ internal class SDKClientManager(
...
+
         directoryPath = ""
         reCheckingCountTime = 0
         commandType = null
         idEdgeDevice = false
         isSaveToOffline = false
         isDebug = false
         isSkipValidation = false
-        brokerType = ""
+        //   brokerType = ""
```
Comment (sw-ot-deveshmevada) [inline IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/SDKClientManager.kt]: remove unwanted code

File: IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/SDKClientManager.kt [hunk 4]
```diff
@@ -252,9 +257,9 @@ internal class SDKClientManager(
                     isSkipValidation = sdkObj.getBoolean("skipValidation")
                 }
 
-                if (sdkObj.has("brokerType")) {
+                /*if (sdkObj.has("brokerType")) {
```
Comment (sw-ot-deveshmevada) [inline IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/SDKClientManager.kt]: remove unwanted code

File: IOTConnectSDK/iotconnect-sdk/build.gradle (hunk 1 above)
Comment (sw-tt-tusharojha) [reply IOTConnectSDK/iotconnect-sdk/build.gradle]: @sw-ot-deveshmevada 
I have changed the version but it is of old iot connect sdk i.e. 1.0 remain commented only when we create library for new iot connect sdk i.e. 2.0

File: IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/SDKClientManager.kt (hunk 3 above)
Comment (sw-tt-tusharojha) [reply IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/SDKClientManager.kt]: @sw-ot-deveshmevada 
Remove all commented code

File: IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/SDKClientManager.kt (hunk 4 above)
Comment (sw-tt-tusharojha) [reply IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/SDKClientManager.kt]: @sw-ot-deveshmevada 
Remove all commented code

File: IOTConnectSDK/iotconnect-sdk_2_1/build.gradle (hunk 2 above)
Comment (sw-tt-tusharojha) [reply IOTConnectSDK/iotconnect-sdk_2_1/build.gradle:110]: @sw-ot-deveshmevada 
These code is used to generate library for different product flavors (AZ/AWS). At a time one is commented and other is used for creating different library according to product flavors (AZ/AWS)

File: README.md [hunk 5]
```diff
@@ -268,10 +266,19 @@ Note : (*) indicates the mandatory element of the object.
...
+	       implementation 'com.github.avnet-iotconnect:iotc-android-sdk:12.2.1-azTest'
+	}
+```
+
+- For AWS Platform
 
 ```java	
 	dependencies {
-	        implementation 'com.iotconnectsdk:iotconnectpoc:3.1.2'		
+	      implementation 'com.github.avnet-iotconnect:iotc-android-sdk:12.2.1-awsTest'
```
Comment (sw-ot-deveshmevada) [inline README.md:281]: Why there is a **Test** at the end?

File: README.md [hunk 6]
```diff
@@ -268,10 +266,19 @@ Note : (*) indicates the mandatory element of the object.
 ```
 
 2.Add below dependency in build.gradle file.
+- For AZ Platform
+  
+```java	
+	dependencies {
+	       implementation 'com.github.avnet-iotconnect:iotc-android-sdk:12.2.1-azTest'
```
Comment (sw-ot-deveshmevada) [inline README.md:273]: Why there is **Test** at the end?

File: IOTConnectSDK/iotconnect-sdk_2_1/build.gradle (hunk 2 above)
Comment (sw-ot-deveshmevada) [reply IOTConnectSDK/iotconnect-sdk_2_1/build.gradle:110]: Can we do like this?
`android {

    flavorDimensions "version"

    productFlavors {
        free {
            dimension "version"
            // Customize properties for the 'free' flavor
            applicationId "com.example.free"
            versionCode 1
            versionName "1.0-free"
        }

        pro {
            dimension "version"
            // Customize properties for the 'pro' flavor
            applicationId "com.example.pro"
            versionCode 1
            versionName "1.0-pro"
        }

        demo {
            dimension "version"
            // Customize properties for the 'demo' flavor
            applicationId "com.example.demo"
            versionCode 1
            versionName "1.0-demo"
        }
    }

}
`

File: IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/webservices/responsebean/IdentityServiceResponse.kt [hunk 7]
```diff
@@ -9,100 +9,100 @@ data class IdentityServiceResponse(
...
-            val r: Int, // 0 ---If 1 – Edge Device can send 203 message to get all rules
+            val r: Int,
             @SerializedName("set")
-            val set: Int // 0 -- If 1 – Device can send 202 message to get updates on settings/twins
+            val set: Int
         )
 
         data class Meta(
             @SerializedName("at")
-            val at: Int, // 1---  Authentication type of device.
```
Comment (sw-ot-deveshmevada) [inline IOTConnectSDK/iotconnect-sdk_2_1/src/main/java/com/iotconnectsdk/webservices/responsebean/IdentityServiceResponse.kt:37]: why did you removed code commenting?

**Review by sw-ot-deveshmevada — COMMENTED** [review]
@sw-tt-tusharojha  I have shared feedback please review

File: IOTConnectSDK/iotconnect-sdk_2_1/build.gradle (hunk 2 above)
Comment (sw-tt-tusharojha) [reply IOTConnectSDK/iotconnect-sdk_2_1/build.gradle:110]: @sw-ot-deveshmevada 
These is used to create product flavor which is done already

**Review by sw-ms-imranbhadelia — APPROVED** [review]

**Pull request merged**